pkg/jobrunaggregator/jobruntablecreator/cmd.go

```diff
@@ -0,0 +1,93 @@
+package jobruntablecreator
+
+import (
+	"context"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"github.com/openshift/ci-tools/pkg/jobrunaggregator/jobrunaggregatorlib"
+)
+
+type BigQueryJobRunTableCreateFlags struct {
+	DataCoordinates *jobrunaggregatorlib.BigQueryDataCoordinates
+	Authentication  *jobrunaggregatorlib.GoogleAuthenticationFlags
+}
+
+func NewBigQueryJobRunTableCreateFlags() *BigQueryJobRunTableCreateFlags {
+	return &BigQueryJobRunTableCreateFlags{
+		DataCoordinates: jobrunaggregatorlib.NewBigQueryDataCoordinates(),
+		Authentication:  jobrunaggregatorlib.NewGoogleAuthenticationFlags(),
+	}
+}
+
+func (f *BigQueryJobRunTableCreateFlags) BindFlags(fs *pflag.FlagSet) {
+	f.DataCoordinates.BindFlags(fs)
+	f.Authentication.BindFlags(fs)
+}
+
+func NewBigQueryJobRunTableCreateFlagsCommand() *cobra.Command {
+	f := NewBigQueryJobRunTableCreateFlags()
+
+	cmd := &cobra.Command{
+		Use:          "create-tables",
+		Short:        "Create Jobs table in bigquery",
+		Long:         "Create Jobs table in bigquery",
+		SilenceUsage: false,
```

Comment:
Looking for feedback on if this should be silent or not.

Reply:
Looking at the docs on that flag it looks largely irrelevant, but I would say false is what we want. False is also the zero (or default) value for a bool in go so you can just technically just omit this field. (but not a required change)

Reply:
When set to `false`, the help screen will show the `create-tables` command; I guess there's no reason to hide it.